emissionsapi/web.py

```diff
@@ -82,21 +83,30 @@ def decorated(*args, **kwargs):
                 kwargs['wkt_polygon'] = polygon_to_wkt(polygon)
             except RESTParamError as err:
                 return str(err), 400
+        # parse paramter point
+        elif point is not None:
+            try:
+                logger.debug('Try to parse point')
+                kwargs['point'] = f'POINT({point[0]} {point[1]})'
+            except KeyError:
+                return 'Invalid point', 400
         return f(*args, **kwargs)
     return decorated
 
 
 @parse_wkt_polygon
 @parse_date('begin', 'end')
 @emissionsapi.db.with_session
-def get_data(session, wkt_polygon=None, begin=None, end=None, limit=None,
-             offset=None):
+def get_data(session, wkt_polygon=None, point=None, begin=None, end=None,
```

Comment:
I'm not sure there is any real reason to keep both parameters and I would probably collapse them to one `wkt` parameter. If you want to keep both  (why?) the parameters should either be `point` and `polygon` or `wkt_point` and `wkt_polygon`.

Reply:
Now there is only a `wkt` and a `distance` parameter.